### ISSUE TITLE
setup pingdom to run monitor status page nightly

### ISSUE THREAD

**Issue by elrayle** [cul-it/qa_server_aws_deploy#5]
The current production site has a pingdom script to load the page after 2:00am.  This load will fail, but starts the process of running all tests.  The script loads the page again after an hour (maybe hourly).  This load and all other loads on that day should pass.

The script can be the same used for the current production site.

**Comment by gdelisle** [comment]
Yes, Pingdom runs every hour. This isn't efficient and puts undue strain on the service because it's forced to run all tests every hour, so we should come up with a better system eventually, but that's another issue.

**Comment by elrayle** [comment]
It does not run the tests every hour.  It loads the page every hour.  The application only runs the tests once a day unless given a token that forces the tests to run again.  The results are cached.  Every page load after the test run uses the cached values.

**Comment by gdelisle** [comment]
I've created a new Pingdom monitor pointed at qa-server-service.library.cornell.edu which will be the non-public service URL. That will work now, and should continue to work post-go-live. At that point, the old Pingdom monitor will also point to the new service (at the old address) and we can disable one of them.

**Comment by elrayle** [comment]
I received a message from Pingdom that the monitor status page is down, and it is.  So it correctly reported the status.  I am going to leave this open until the status page is fixed and we can confirm that Pingdom correctly detects that the page is working.  Probably no more work to be done on this issue.

**Comment by elrayle** [comment]
Close once this is confirmed to be working when the switch over is complete.
